libs/application/ui-fields/src/lib/CheckboxFormField.tsx

```diff
@@ -17,14 +18,17 @@ const CheckboxFormField: FC<Props> = ({
   field,
   formValue,
 }) => {
-  const { id, name, options, disabled } = field
+  const { id, name, description, options, disabled } = field
   const { formatMessage } = useLocale()
 
   return (
     <div>
       {showFieldName && (
         <Typography variant="p">{formatMessage(name)}</Typography>
       )}
+
+      {description && <Description description={formatMessage(description)} />}
```

Comment:
I am curious how a field with description in a multifield will look

Reply:
Good point, I assume it would be designed for only one description, so they could leave the second field description blank. Will keep it in mind if we want to enhance.